### DISCUSSION TITLE
Lowest possible latency setting?

### DISCUSSION THREAD

**Discussion by alatteri** [Haivision/srt#2532] [Q&A] [answered]
Hello,

If I have an endpoint who's LINK RTT is below 2s... what is the lowest support latency setting with srt-live-transmit?  Can I go as low as 6ms?  I believe there used to be a hard floor prior to v1.5 that was around 30ms.

**Comment by maxsharabayko** [comment, marked as answer]
Yes, you can go to 6 ms. Starting from v1.5.1 (with the new receiver buffer implementation) the lowest possible is 1 ms, because 0 is a special value for the default latency of 120 ms.
See [SRTO_LATENCY](https://github.com/Haivision/srt/blob/v1.5.1/docs/API/API-socket-options.md#SRTO_LATENCY).
Still, the lower you go, the more chances some packet loss remains unrecovered. The recommended latency value for wired networks is 3 to 4 times RTT.

**Comment by alatteri** [comment]
Thank you for the response.  This makes sense.   Much appreciated.
